testkit/src/main/scala/org/bitcoins/testkit/wallet/DLCWalletUtil.scala

```diff
@@ -101,4 +107,45 @@ object DLCWalletUtil {
 
   lazy val sampleDLCSign: DLCSign =
     DLCSign(dummyCETSigs, dummyFundingSignatures, sampleDLCEventId)
+
+  def initDLC(fundedWalletA: FundedWallet, fundedWalletB: FundedWallet)(
+      implicit ec: ExecutionContext): Future[
+    (InitializedDLCWallet, InitializedDLCWallet)] = {
+    val walletA = fundedWalletA.wallet
+    val walletB = fundedWalletB.wallet
+
+    val sampleOfferData = sampleDLCOffer
+    for {
+      offer <- walletA.createDLCOffer(
+        sampleOfferData.oracleInfo,
+        sampleOfferData.contractInfo,
+        Satoshis(6000),
+        Some(sampleOfferData.feeRate),
+        sampleOfferData.timeouts.contractMaturity.toUInt32,
+        sampleOfferData.timeouts.contractTimeout.toUInt32
+      )
+      accept <- walletB.acceptDLCOffer(offer)
+      sigs <- walletA.signDLC(accept)
+      _ <- walletB.addDLCSigs(sigs)
+    } yield {
+      (InitializedDLCWallet(FundedWallet(walletA)),
+       InitializedDLCWallet(FundedWallet(walletB)))
+    }
+  }
+}
+
+object DLCWalletUtil extends DLCWalletUtil {
+
+  case class InitializedDLCWallet(funded: FundedWallet) {
+    val wallet: Wallet = funded.wallet
+  }
+
+  def createDLCWallets(
+      fundedWalletA: FundedWallet,
+      fundedWalletB: FundedWallet)(implicit system: ActorSystem): Future[
```

Comment:
From what I see here you probably want an implicit `ExecutionContext` rather than a full-fledged `ActorSystem`